### ISSUE TITLE
Raspberry 3B + 

### ISSUE THREAD

**Issue by Vay1** [CIRCL/Circlean#75]
Hello, 

It doesn't work on my raspberry 3B + , is it normal ?

**Comment by Rafiot** [comment]
No, it should work. What is not working?

**Comment by Vay1** [comment]
i used this image 2020-01-16_CIRCLean.img.gz but when i try with 2 usb groomer tell me that it doesn't find the path of some folder

**Comment by Rafiot** [comment]
what is the file format of the USB keys?

**Comment by Vay1** [comment]
both are FAT32

**Comment by Rafiot** [comment]
ok, that should work fine then.

Any chance you can take a picture of the error message? That sounds like one of the key cannot be mounted or something like that. I've seen it in the past, sometimes, some keys take a very long time to be mounted and it fails. 
In theory, if the destination key can be mounted, there should at least be the begining of a logfile.

**Comment by Vay1** [comment]
Ok I will show you but, i'm not at work right now so i will sent you images here when I can. It shouldn't be too long max 20 min.

**Comment by Rafiot** [comment]
No rush, I'm not sure I'll have time to work on it today.

**Comment by Vay1** [comment]
![DSC_0243](https://user-images.githubusercontent.com/64682996/124436773-9e7fff00-dd76-11eb-9349-82323a26645c.JPG)
so this is the screen where groomer tell me he doesn't find some folder.

**Comment by Rafiot** [comment]
yeah, but it also says `/org/freedesktop/UDisks2/block_devices/sda` isn't a mountable filesystem, and that should be mounted on `/dev/source_key`.

Is there more than one partition on the source key? It should be supported fine, but I think the path should be `/org/freedesktop/UDisks2/block_devices/sda1` so that's weird.

**Comment by Vay1** [comment]
Both USB has just one partition

**Comment by Rafiot** [comment]
That sounds like a very standard case, I'm not sure what's going on there.

You have no problem mounting either keys on an other machine? And before removing the keys from the other machine, you do a clean unmount? 

At this point, I'd try with an other source key, just to check? I saw keys in the past that are just not working on a rPi, but in that case, the rPi doesn't fund the keys at all and it fails differently...

**Comment by Vay1** [comment]
I can mount both USB on my computer there is no problem and I always do clean unmount so I don't know. OK i will try with 2 other USB to see if something is different

**Comment by Vay1** [comment]
So i tried with other USB and it's working I think because i have this : 
![DSC_0244](https://user-images.githubusercontent.com/64682996/124465305-8d92b600-dd95-11eb-8603-454b390cfd7d.JPG)
but in the end of the processus I got this  and there is no documents exported on my clean USB from the one not clean
![DSC_0247](https://user-images.githubusercontent.com/64682996/124465416-ae5b0b80-dd95-11eb-9e48-d28b29807cbf.JPG)

**Comment by Rafiot** [comment]
Yeah, it seems it worked fine, the process seems to finish fine. And there was absolutely nothing on the destination key (the one plugged anywhere but the top left port), not even logs?

**Comment by Vay1** [comment]
no there is nothing this is weird

**Comment by Rafiot** [comment]
Yeah, I cannot reproduce that. 

I see the under voltage messages. Can you try to plug the rpi to something providing a higher voltage?

**Comment by Vay1** [comment]
there is no change with a higher voltage

**Comment by Vay1** [comment]
Ok I found the problem, it was my USB which wasn't big enough in storage capacity. thanks for helping me.

**Comment by Rafiot** [comment]
oh, that's super weird... I need to add a warning for that somewhere.

**Comment by Vay1** [comment]
What do you use to analyze the cle usb, there is an anti virus or something else ?

**Comment by Rafiot** [comment]
No antivirus, the application search for active content in the files and mark them as malicious in case there is something phishy.

The library is there: https://github.com/CIRCL/PyCIRCLean And this it the file that does the work: https://github.com/CIRCL/PyCIRCLean/blob/main/filecheck/filecheck.py

**Comment by piep-piep-piep** [comment]
I ran into this aswell, on 3B+ the USB-port assignment seems to be wrong. This causes circlean to fail when using it according to the userguide (top-left as source, other port, for me bottom left for destination). I tried to plug the keys into different ports and managed to get the files to be copied (just not the way aroud i expected...).
[This pull request](https://github.com/CIRCL/Circlean/pull/72) seems like a reasonable fix for the problem, could this be merged at some point? (i see that there is a merge conflict, but cant see the details..)
